infracost-usage-example.yml

```diff
@@ -575,6 +575,14 @@ resource_usage:
   google_pubsub_topic.my_topic:
     monthly_message_data_tb: 7.416 # Monthly amount of message data published to the topic in TB.
 
+  google_secret_manager_secret.my_secret:
+    active_secret_versions: 10000       # Number of active secret versions in each month. NOTE: this is used only when secret versions are not defined.
```

Comment:
@aliscott Added a NOTE to explain the usage.

Reply:
👍